src/mcts/node.cc

```diff
@@ -332,9 +332,10 @@ void Node::FinalizeScoreUpdate(float v, float d, int multivisit) {
           const auto n = child.GetNBetamcts();
           const auto r = child.edge()->GetRBetamcts();
```

Comment:
move this inside the conditional?

Reply:
I think you can't declare variables within conditionals, even when you only use them within the conditional? At least when I tried it the compiler didn't like it...

Reply:
I meant inside the body of the condition.
'         if (n > 0) {
            **const auto r = child.GetRBetamcts();** 
            const auto visits_eff = r * n; 
            n_temp += visits_eff; 
            // Flip Q for opponent. 
            q_temp += -child.node()->GetQBetamcts() * visits_eff; 
          }'